pcsx2/DebugTools/DebugInterface.cpp

```diff
@@ -348,7 +348,9 @@ int R5900DebugInterface::getRegisterCount(int cat)
 	case EECAT_CP0:
 	case EECAT_FPR:
 	case EECAT_FCR:
+		return 32;
 	case EECAT_VU0F:
+		return 33;  // 32 + ACC
```

Comment:
IMO the way you originally had it was better.

Reply:
Will it cause any issues if it's like this ? I still prefer this way since it's consistent but if functionality wise the other was better then the second commit can be dropped.

Reply:
The other functions return different values for each case, so in that case I prefer enum ordering.

This function however returns the same value for multiple cases. In this case I prefer having the cases that return the same values grouped together. I can determine EECAT_GPR has 35 rows, EECAT_VU0F has 33 rows and everything else has 32 rows that little bit faster. `getRegisterSize()` and `getRegisterType()` also do the same thing.